### PR TITLE
add format team enderdragon name

### PR DESCRIPTION
**Pull request by ImYenil**
add format team enderdragon name

### CODE REVIEW

File: versionsupport_1_12_r1/src/main/java/com/tomkeuper/bedwars/support/version/v1_12_R1/v1_12_R1.java [hunk 1]
```diff
@@ -329,7 +331,11 @@ public void spawnDragon(Location l, ITeam bwt) {
             return;
         }
         EnderDragon ed = (EnderDragon) l.getWorld().spawnEntity(l, EntityType.ENDER_DRAGON);
-        ed.setPhase(EnderDragon.Phase.CIRCLING);
+        for (Player p : bwt.getMembers()) {
+            ed.setCustomName(getMsg(p, Messages.FORMAT_TEAM_ENDERDRAGON_NAME).replace("%bw_team_color%", bwt.getColor().chat().toString()).replace("%bw_team_name%", bwt.getDisplayName(Language.getPlayerLanguage(p))));
+            ed.setCustomNameVisible(true);
+            ed.setPhase(EnderDragon.Phase.CIRCLING);
```
Comment (tomkeuper) [inline versionsupport_1_12_r1/src/main/java/com/tomkeuper/bedwars/support/version/v1_12_R1/v1_12_R1.java:337]: How does this set the name for each player?

Comment (ImYenil) [reply versionsupport_1_12_r1/src/main/java/com/tomkeuper/bedwars/support/version/v1_12_R1/v1_12_R1.java:337]: > How does this set the name for each player?

Set this to the team dragon name for each team.

Comment (tomkeuper) [reply versionsupport_1_12_r1/src/main/java/com/tomkeuper/bedwars/support/version/v1_12_R1/v1_12_R1.java:337]: No, first of all you are setting  the phase of the enderdragon to the same value for every player, 1 time is enough.
second, setting the custom name is the same for everyone as you are not getting a personalized enderdragon instance, dont even know if thats possible tbh. 
third you are just overruling the value of the previous loop it looks like

Comment (ImYenil) [reply versionsupport_1_12_r1/src/main/java/com/tomkeuper/bedwars/support/version/v1_12_R1/v1_12_R1.java:337]: > No, first of all you are setting the phase of the enderdragon to the same value for every player, 1 time is enough. second, setting the custom name is the same for everyone as you are not getting a personalized enderdragon instance, dont even know if thats possible tbh. third you are just overruling the value of the previous loop it looks like

In this case, then how do you get the Player variable?

Comment (tomkeuper) [reply versionsupport_1_12_r1/src/main/java/com/tomkeuper/bedwars/support/version/v1_12_R1/v1_12_R1.java:337]: Like I said I dont know if you can at al

Comment (ImYenil) [reply versionsupport_1_12_r1/src/main/java/com/tomkeuper/bedwars/support/version/v1_12_R1/v1_12_R1.java:337]: > Like I said I dont know if you can at al

I don't know. then you make it

**Pull request closed without merging**